lib/storage/adapters/embedded.js

```diff
@@ -205,7 +219,8 @@ EmbeddedStorageAdapter.prototype._size = function(key, callback) {
 
       /* istanbul ignore if */
       if (key) {
-        self._fs.stat(key, handleStatResults);
+        self._fs.stat(crypto.createHash('sha1').update(key).digest('hex'),
```

Comment:
For the purpose of keeping consistent with the hash function used elsewhere in storj-lib ([and the recent discovery that sha1 is dead](http://shattered.io/)), let's replace this with:

```
utils.rmd160(key, 'hex');
```